taco-team-build.js

```diff
@@ -90,7 +90,9 @@ function applyExecutionBitFix(platforms) {
         var platformCordovaDir = "platforms/" + platform + "/cordova";
         
         if (utilities.fileExistsSync(platformCordovaDir)) {
-            script += "find -E " + platformCordovaDir + " -type f -regex \"[^.(LICENSE)]*\" -exec chmod +x {} +\n"
+            // Disable -E flag for non-OSX platforms
+            var regex_flag = process.platform == 'darwin' ? '-E' : '';
```

Comment:
The check directly above exits the function if we're not on OSX, so right now this additional check won't do anything. We would need to alter the check at the top of the function to continue for both OSX *and* Linux.